### DISCUSSION TITLE
How Can I Install Git for Unity

### DISCUSSION THREAD

**Discussion by rex644** [spoiledcat/git-for-unity#14] [General]
How do I install for 2021.3? The install screenshots do not match up. Would we install with the GitURL in Unity maybe? That gives me an error:
[Package Manager Window] Cannot perform upm operation: Unable to add package [https://registry.spoiledcat.com]:
  Package name 'https://registry.spoiledcat.com' is invalid. [InvalidParameter].
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()

Git URL I tried: https://github.com/spoiledcat/git-for-unity

![image](https://user-images.githubusercontent.com/8412603/211925405-b17e523b-e7a9-45d9-a87f-8be9a404356b.png)

![image](https://user-images.githubusercontent.com/8412603/211925546-6158a2db-bf09-4ee8-938c-090c0341ba43.png)

**Comment by deertwoheads** [comment]
Rather than in the Package Manager itself, you need to go to Project Settings > Package Manager and add the registry there. Then, if Unity decides you are worthy, a "My Registries" option will appear in the Package Manager "Packages" menu.

Unfortunately, it seems Unity has a bug where sometimes this menu simply does not appear, and I have absolutely no idea what the solution is. Perhaps a blood offering?
